### PR TITLE
backstage: extract true-sass test runner from obt and into it's own tool

### PR DESCRIPTION
**Pull request by JakeChampion**
One less thing to use obt for :D

### CODE REVIEW

File: scripts/component/test.bash [hunk 1]
```diff
@@ -2,4 +2,5 @@
 set -e
 origami-build-tools test
 a11y
-validate-component-demo-html
\ No newline at end of file
+validate-component-demo-html
+test-sass
```
Comment (chee) [inline scripts/component/test.bash:6]:
```suggestion
test-sass

```

File: tools/test-sass/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import {globbySync as glob} from "globby"
+const sassTestFiles = glob('./test/**/index.test.scss', {onlyFiles: true})
+if (sassTestFiles.length) {
+  try {
+    await $`mocha ../../tools/test-sass/sass-test-suite.js`
+  } catch {
+    process.exitCode = 1;
+  }
+}
```
Comment (chee) [inline tools/test-sass/index.js:13]:
```suggestion
}

```

File: tools/test-sass/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+import {$} from "zx"
+
+import {globbySync as glob} from "globby"
+const sassTestFiles = glob('./test/**/index.test.scss', {onlyFiles: true})
+if (sassTestFiles.length) {
+  try {
+    await $`mocha ../../tools/test-sass/sass-test-suite.js`
```
Comment (chee) [inline tools/test-sass/index.js:9]: i don't know why, but this particular `../../` seems confusing to me. probably fine though.

**Review by chee — APPROVED** [review]

File: tools/test-sass/index.js (hunk 3 above)
Comment (JakeChampion) [reply tools/test-sass/index.js:9]: I think we could change this to using __dirname instead

Comment (JakeChampion) [reply tools/test-sass/index.js]: @chee I changed it to use __dirname and it works -- [`1854e61` (#397)](https://github.com/Financial-Times/origami/pull/397/commits/1854e61959e419e4ed2429dc8bb5581a26063dc0)

File: tools/test-sass/index.js [hunk 4]
```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/env node
 
-import {$} from "zx"
+import {$} from 'zx'
+import {globbySync as glob} from 'globby'
+import { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
```
Comment (chee) [inline tools/test-sass/index.js:6]: nice :)))

**Pull request merged**